app/services/fetch_oembed_service.rb

```diff
@@ -100,7 +100,7 @@ def parse_for_format(body)
   end
 
   def validate(oembed)
-    oembed if oembed[:version].to_s == '1.0' && oembed[:type].present?
+    oembed if oembed.present? && oembed[:version].to_s == '1.0' && oembed[:type].present?
```

Comment:
Does this handle the scenario where a response body was present ... but was not parsable in the prior `Oj.load` step?